### DISCUSSION TITLE
Serialization Warning

### DISCUSSION THREAD

**Discussion by yx1215** [awslabs/gluonts#1648] [Q&A] [answered]
When I used predictor.serialize with SimpleFeedForward, DeepAR and MQCNN, I would get the following warning:
```
Serializing RepresentableBlockPredictor instances 
does not save the prediction network structure in a backwards-compatible manner. 
Be careful not to use this method in production.
```
What is the reason for causing this warning? If I deserialize the saved files, what function would be different from the one before serialize? How should we avoid this warning?

**Comment by mbohlkeschneider** [comment, marked as answer]
Hi @yx1215,

This means that there is no guarantee that your old saved models will still load when you update GluonTS. In other words, GluonTS updates are not tested for backwards compatibility of saved models (it is simply not feasible with all the models that are currently in GluonTS). 

Hope this was clear. Let me know if there are further questions.

**Reply by yx1215** [reply]
It seems that this would happen to most of the packages online? It would be better if an internal way to suppress this warning message is provided.
